src/main/java/org/recordrobotics/charger/subsystems/Claw.java

```diff
@@ -0,0 +1,36 @@
+package org.recordrobotics.charger.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import org.recordrobotics.charger.RobotMap;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+public class Claw extends SubsystemBase {
+	private static final double GEAR_RATIO = 63;
+
+	public static final double CLAW_NEUTRAL = 0.0;
+	public static final double CLAW_CUBE = 0.2;
+	public static final double CLAW_CONE = 0.3;
+
+	private CANSparkMax _motor = new CANSparkMax(RobotMap.Claw.CLAW_MOTOR_PORT, MotorType.kBrushless);
+
+	public Claw(){
+		_motor.set(0);
+		_motor.getEncoder().setPositionConversionFactor(1 / GEAR_RATIO);
+	}
+
+
+	public void turn(double speed){
+		_motor.set(Subsystems.limitSpeed(speed));
+	}
+
+	public double getPosition(){
+		return _motor.getEncoder().getPosition()/GEAR_RATIO;
```

Comment:
i believe this was accidentally left here. you don't need to divide by gear ratio again, as `getPosition()` returns the actual encoder value times the parameter in `setPositionConversionFactor()`